### ISSUE TITLE
well-trained model fails at training data

### ISSUE THREAD

**Issue by johnbroughton2017** [apache/incubator-mxnet#12707]
Hi all,

For some reason I am encountering the following problem:
I trained an alexnet type model until the training accuracy is about 100%, which is the output of the training process. To validate the model, I manually load the last checkpoint and then predict on the training data. But the accuracy drops to about 80%. I think something must be wrong. I tried adding or removing the mean RGB but it seems the same. 

Anyone with experience on this issue? Any suggestion is highly appreciated. Thanks.

-- J

PS: I followed this tutorial to load pretrained model
https://mxnet.incubator.apache.org/tutorials/python/predict_image.html

**Comment by vandanavk** [comment]
@mxnet-label-bot [Question, Example]

**Comment by johnbroughton2017** [comment]
nvm.

seems to be making some mistakes during color conversion!
